priv/templates/phx.gen.auth/confirmation_live_test.exs

```diff
@@ -33,7 +33,7 @@ defmodule <%= inspect context.web_module %>.<%= inspect Module.concat(schema.web
 
       assert {:ok, conn} = result
       assert Phoenix.Flash.get(conn.assigns.flash, :info) =~
-        "User confirmed successfully"
+        "<%= inspect schema.alias %> confirmed successfully"
```

Comment:
It feels like this should match what the developer is naming the "user" as the UI probably matches their domain.